templates/front/my_checks_desktop.html

```diff
@@ -12,6 +12,7 @@
         <th></th>
     </tr>
     {% for check in checks %}
+    {% if check.id in allowed_checks or check.user_id == user.id %}
```

Comment:
We are doing this in order to get both the members checks and the checks that they are allowed to view from their respective teams. This is because the member has two 'categories' of checks.